src/main/java/net/sacredlabyrinth/Phaed/PreciousStones/PreciousStones.java

```diff
@@ -6,14 +6,10 @@
 import net.sacredlabyrinth.Phaed.PreciousStones.listeners.*;
 import net.sacredlabyrinth.Phaed.PreciousStones.managers.*;
 import net.sacredlabyrinth.Phaed.PreciousStones.uuid.UUIDMigration;
-import org.bukkit.ChatColor;
 import org.bukkit.plugin.java.JavaPlugin;
 
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.net.URL;
-import java.nio.charset.StandardCharsets;
+import com.sk89q.worldedit.bukkit.WorldEditPlugin;
```

Comment:
I can tell you right off the bat that I can't pull this, sorry.

This is hard depends on WorldEdit!

Reply:
Seems like it might not be too hard to work around though, just need to get rid of this reference .. see below